.codesandbox/ci.json

```diff
@@ -0,0 +1,10 @@
+{
+  "packages": ["packages/react", "packages/react-dom"],
+  "buildCommand": "build --type=NODE react/index,react-dom/index,scheduler/index",
+  "publishDirectory": {
+    "react": "build/node_modules/react",
+    "react-dom": "build/node_modules/react-dom",
+    "scheduler": "build/node_modules/scheduler"
```

Comment:
I think this will cause the same issue as https://github.com/mui-org/material-ui/pull/17976#issuecomment-546294181:
`react-dom` declares a dependency on `scheduler` with the newly created version but codesandbox thinks this version is on the npm registry.

I guess this applies regardless of whether `scheduler` is built in the codesandbox CI or not.